### PR TITLE
Unconfuse checksum for gorry

### PR DESCRIPTION
**Pull request by gorryfair**
I was so confused, if people like this PR I think I might be less confused in future.

### CODE REVIEW

**Comment by gorryfair** [comment]
Thought to fix #776

**Comment by gorryfair** [comment]
PR for issue: #776

File: draft-ietf-taps-interface.md [hunk 1]
```diff
@@ -1776,14 +1776,15 @@ Name:
...
 
 Default:
 : Full Coverage
 
 This property specifies the minimum number of bytes in a received
 message that need to be covered by a checksum. A special value of 0 means
-that no checksum is permitted. A receiving Endpoint will not forward messages to the application
+that a received packet does not need to have a non-zero checksum field. 
+A receiving Endpoint will not forward messages to the application
 that have less coverage. The application is responsible for handling
```
Comment (mwelzl) [inline draft-ietf-taps-interface.md:1788]: Shouldn't this be "A receiving Endpoint will not forward messages that have less coverage to the application." ?

**Review by mwelzl — APPROVED** [review]
Thanks - great catch about s/perMsgChecksumLenSend/perMsgChecksumSend  (and -Recv) in particular, these must have been historical remnants.

I added one comment related to English, take a look to see if you agree.

**Pull request merged**